### ISSUE TITLE
Questions about the dataset used for training MLP and planning routes

### ISSUE THREAD

**Issue by junsu-kim97** [binghong-ml/retro_star#11]
I am thankful for this interesting work and the shared code!

I have questions about the dataset for training MLP and the planning routes in Retro*.

1) The train/val/test routes are extracted from USPTO-full train/val/test, which is at GLN github repo? (https://github.com/Hanjun-Dai/GLN)

2) The train/val/test dataset for training MLP is same as the train/val/test dataset for building routes?

In summary, I want to ask you that whether "data split at GLN == data split for building routes == data split for training MLP" is right.

Thanks in advance.

Junsu Kim

**Comment by binghong-ml** [comment]
Hi Junsu, thanks for the interest in our work.

Retro* assumes a well-trained one-step model. It can be simple models like MLP or more advanced ones like GLN.

In our experiment, to simulate real-world scenarios, the training split for one-step MLP is the same as the training split for routes. To answer your question in short, "data split at GLN != data split for routes == data split for training MLP".

Binghong

**Comment by binghong-ml** [comment]
Please refer to http://binghongchen.net/pdf/ICML-retrosyn-slide.pdf page 14 for illustration on dataset construction. Thanks!

**Comment by JunsuKim-Kaist** [comment]
Thanks for the answer!

I'd like to ask you whether you can offer the raw dataset(train/val/test) for training and evaluation MLP (==for building routes)

Best,
Junsu Kim

**Comment by junsu-kim97** [comment]
Thanks for the answers and helpful slides!
